roadmap.md

```diff
@@ -0,0 +1,92 @@
+
+- [x] Some items from the previous scenario
+- [ ] [Network events for the request to be sent, and response started and completed](https://github.com/w3c/webdriver-bidi/pull/204)
+
+### Submitting forms
+
+This scenario loads a web page, enters text into a form field via the keyboard, and submits the form via a mouse click before extracting the results from the page. In spec terms, this involves:
+
+- [x] Everything from the previous scenario
+- [ ] [Emulating keyboard input](https://github.com/w3c/webdriver-bidi/pull/175)
+- [ ] [Emulating mouse input](https://github.com/w3c/webdriver-bidi/pull/175)
+
+### Observing changes being made to the DOM tree
+
+In this scenario a MutationObserver will be installed by a bootstrap script as early as the document gets created. It watches for changes made to the DOM tree and sends the relevant updates to the client. In spec terms, this involves:
+
+- [x] Some items from the previous scenarios
+- [ ] [Adding a bootstrap script](https://w3c.github.io/webdriver-bidi/#command-script-addPreloadScript)
+- [ ] [Installing the bootstrap script](https://w3c.github.io/webdriver-bidi/#preload-scripts)
+- [ ] [Back channel for communicating with the client](https://github.com/w3c/webdriver-bidi/pull/361)
+- [ ] [Removing a bootstrap script][https://w3c.github.io/webdriver-bidi/#command-script-removePreloadScript]
+
+### Replacing resources with test data
+
+This scenario loads a web page and uses network request interception to replace any image in that page with a custom image. In spec terms, this involves:
+
+- [x] Some items from the previous scenarios
+- [ ] [Intercepting network requests](https://github.com/w3c/webdriver-bidi/issues/66)
+
+### HTTP authentication
+
+This scenario loads a web page that is protected behind user credentials. In spec terms, this involves:
+
+- [x] Some items from the previous scenarios
+- [ ] [The event for a HTTP auth challenge](https://github.com/w3c/webdriver-bidi/issues/66)
+- [ ] [The command to provide the authentication response](https://github.com/w3c/webdriver-bidi/issues/66)
+
+### Handling onbeforeunload prompts
+
+This scenario loads a web page with a registered `beforeunload` event handler. After updating the value of some form input elements it should be checked that navigating away opens the beforeonload prompt. In spec terms, this involves:
+
+- [x] Some items from the previous scenarios
+- [ ] [The event when a user prompt opens](https://w3c.github.io/webdriver-bidi/#webdriver-bidi-user-prompt-opened)
+- [ ] [Handling the beforeunload prompt](https://w3c.github.io/webdriver-bidi/#command-browsingContext-handleUserPrompt)
+- [ ] [The event when a user prompt closes](https://w3c.github.io/webdriver-bidi/#webdriver-bidi-user-prompt-closed)
+
+### Capturing screenshots
```

Comment:
I'd move this to after "submitting forms", since it's basically a prerequisite for many kinds of testing.